web_src/js/features/repo-migrate.js

```diff
@@ -59,6 +58,7 @@ export function initRepoMigrationStatusChecker() {
 }
 
 async function doMigrationRetry(e) {
-  await POST($(e.target).attr('data-migrating-task-retry-url'));
+  const retryUrl = e.target.getAttribute('data-migrating-task-retry-url');
+  await POST(retryUrl);
```

Comment:
```suggestion
  await POST(e.target.getAttribute('data-migrating-task-retry-url'));
```

Guess I prefer this one-line, also because the variable name does not add anything useful.